File: python class 1/suma.py
def sum(a, b):
    while b != 0:
        carry = a & b
        a = a ^ b
        b = carry << 1
    return a

num1 = int(input("Ingresa un número: "))
num2 = int(input("Ingresa otro número: "))
print(sum(num1, num2))

File: others_class/python class 1/suma.js
function add(a, b) {
    while (b != 0) {
        let carry = a & b;
        a = a ^ b;
        b = carry << 1;
    }
    return a;
}

let num1 = process.argv[2];
let num2 = process.argv[3];
console.log(add(num1, num2));